libraries/core-react/stories/components/Table.stories.tsx

```diff
@@ -18,6 +18,28 @@ Icon.add({ chevron_down, chevron_up })
 
 const { Caption, Body, Row, Cell, Head } = Table
```

Comment:
Maybe make a seperate issue for converting existing stories to use dot-style in examples 🤔

Reply:
No, I think we should do it now, doesnt take long. Just didn't think about it 🙃